SMTG-UCL/sumo
# Functionality to plot dielectric loss function

## Description of work

This PR adds a functionality to plot the loss function as determined from the dielectric function. It is heavily based on the optical absorption routines and reads in from a `vasprun.xml`

This addresses issue #22.

## To test

* Run `setup.py install` on this branch of `sumo`
* In `tests/data/Cs2SnBr6` run `sumo-loss` and `sumo-loss -a`; check that results are sensible
* Check that docstrings produce sensible documentation for the api
